Sources/Foundation/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftFoundation open source project
+##
+## Copyright (c) 2024 Apple Inc. and the SwiftFoundation project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of SwiftFoundation project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+file(GLOB_RECURSE _FoundationPreviewSources "*.swift")
```

Comment:
Let's not build the "FoundationPreview" package in this mode.

Reply:
You are aware of the limitation here that if you add a new file or remove a file you will need to do a clean build as the added/removed source will not be reflected in the build.